uportal-war/src/main/java/org/jasig/portal/groups/pags/testers/EagerRegexTester.java

```diff
@@ -55,7 +57,15 @@
  * @see RegexTester
  */
 public class EagerRegexTester extends RegexTester {
-    
+
+    public EagerRegexTester(IPersonAttributesGroupTestDefinition definition) {
```

Comment:
Yep -- sounds good.  I intend to run through both this PR and the next one today looking for places to add @Since, @Deprecated (the annotation, not merely the javadoc bit), and more comments where appropriate.